### DISCUSSION TITLE
Github 2FA SMS Authentication is not supported in Pakistan

### DISCUSSION THREAD

**Discussion by sanamumtaz** [community/community#66959] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Since we need to enable two-factor authentication on our account before September 28, 2023, hence the SMS Authentication should be supported for Pakistan as well.

I found a related [discussion opened in 2022](https://github.com/orgs/community/discussions/15793), but there is no engagement on it.

**Comment by hafiz-muhammad** [comment]
You don't need to rely on just SMS for your GitHub account 2FA, there are other methods such as an authenticator app and security keys.

**Reply by sanamumtaz** [reply]
The support should be rolled out for everyone. SMS authentication is available for Pakistan on many platforms.

**Reply by hafiz-muhammad** [reply, marked as answer]
GitHub only supports two-factor authentication via SMS for certain countries due to delivery success rates. You can refer to this GitHub documentation: [https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/countries-where-sms-authentication-is-supported](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/countries-where-sms-authentication-is-supported)

**Comment by farazahmed** [comment]
We need to enable 2FA for Pakistan as suggested by @sanamumtaz.

Github forcefully enables 2FA while service is not supported in Pakistan. 

If it is enabled by default then how it will work?

**Reply by ZanairHassan** [reply]
in pakistan if the 2FA not supported then did they restrict us to enable the 2FA as in the Country or region code dropdown list view the pakistan did not exist.

**Comment by noumandilshad** [comment]
There are Millions of github users for Pakistan, and still dont have the 2fa via sms.
This is very disappointing.

**Comment by danish-119** [comment]
There are Millions of GitHub users in Pakistan, and they still don't have the 2fa via sms.
This isn't very pleasant.

**Comment by iforibraheem** [comment]
This is very dissappointing

**Comment by tajiknomi** [comment]
It annoying that it doesn't support +92 SMS facility but you can use authenticator app like "Google Authenticator" and scan the QR code to setup the 2FA.

**Comment by ZanairHassan** [comment]
T h a n k s   f o r   y o u r   r e s p o n s e ,   I   f o u n d   t h i
s   h e l p f u l   f o r   m e

 Z a n a i r   H a s s a n
 B a c h e l o r ' s   o f   I n f o r m a t i o n   T e c h n o l o g y
 P h o n e :   + 9 2 3 4 4 1 2 2 9 8 6 7
 G m a i l :     z a n a i r g h u m m a n @ g m a i l . c o m
 L i n k e d I n :   w w w . l i n k e d i n . c o m / i n / z a n a i r h
a s s a n
 F r o m :   t a j i k n o m i
 S e n t :   F r i d a y ,   J u l y   1 9 ,   2 0 2 4   1 1 : 0 0   A M
 T o :   c o m m u n i t y / c o m m u n i t y
 C c :   Z a n a i r   H a s s a n ;   C o m m e n t
 S u b j e c t :   R e :   [ c o m m u n i t y / c o m m u n i t y ]   G i
t h u b   2 F A   S M S   A u t h e n t i c a t i o n   i s   n o t   s u p
p o r t e d   i n   P a k i s t a n   ( D i s c u s s i o n   # 6 6 9 5 9
)

 I t   a n n o y i n g   t h a t   i t   d o e s n ' t   s u p p o r t   +
9 2   S M S   f a c i l i t y   b u t   y o u   c a n   u s e   a u t h e n
t i c a t o r   a p p   l i k e   " G o o g l e   A u t h e n t i c a t o r
"   a n d   s c a n   t h e   Q R   c o d e   t o   s e t u p   t h e   2 F
A .

 R e p l y   t o   t h i s   e m a i l   d i r e c t l y ,   v i e w   i t
 o n   G i t H u b ,   o r   u n s u b s c r i b e .
 Y o u   a r e   r e c e i v i n g   t h i s   b e c a u s e   y o u   c o
m m e n t e d . M e s s a g e   I D :   < c o m m u n i t y / c o m m u n i
t y / r e p o - d i s c u s s i o n s / 6 6 9 5 9 / c o m m e n t s / 1 0 0
9 1 9 5 5 @ g i t h u b . c o m >

**Comment by hariskhurshid52** [comment]
The same happened to me, I'm trying to reach contact support, and they are asking for SMS verification, but Pakistan is not listed there, my account is flagged, I don't know whats the reason, and I cant access customer support either.
